File: Python Excel Examples/CellsApiDemo/row/groupRows.py
import spirecloudexcel
from spirecloudexcel.configuration import Configuration as ExcelConfiguration
from spirecloudexcel.api.cells_api import CellsApi

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = ExcelConfiguration(appId, appKey,baseUrl)
api = spirecloudexcel.api.cells_api.CellsApi(configuration)
name = "GroupRows_1.xlsx"
sheetName = "Sheet4"
first_index = 2
last_index = 6
hide = "true"
storage = ""
folder = "/Cells/Row/"
api.group_rows(name, sheet_name=sheetName, first_index=first_index, last_index=last_index, hide=hide,
                    folder=folder,
                    storage=storage)

File: PHP Excel Examples/CellsApiDemo/Row/groupRows.php
<?php

use Spire\Cloud\Excel\Sdk\Api\CellsApi;
use Spire\Cloud\Excel\Sdk\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey, $baseUrl);
$apiInstance = new CellsApi($configuration);

$name = "GroupRows.xlsx";
$folder = "input";
$storage = null;
$sheetName = "Sheet4";
$firstIndex = 2;
$lastIndex = 6;
$hide = true;
$apiInstance->groupRows($name, $sheetName, $firstIndex, $lastIndex, $hide, $folder, $storage);
